tests/test_template.py

```diff
@@ -80,4 +81,11 @@ def test_render_json_template_invalid_json_missing_value(monkeypatch):
     template_file = StringIO('{"type_of_muffin": {{ MUFFIN_TYPE }}}')
 
     with pytest.raises(ValueError):
+        render_json_template(template_file, **{"MUFFIN_TYPE": ""})
+
+
+def test_render_json_template_missing_value_raises(monkeypatch):
```

Comment:
I'm fine without it. I feel like it would be very hard to get one to fail without causing the other to fail in that case, I'm not a fan of a single failure failing multiple specs if I can avoid it. LGTM.